src/common/components/nav/navCartIcon.component.js

```diff
@@ -9,8 +9,9 @@ const componentName = 'navCartIcon'
 
 class NavCartIconController {
   /* @ngInject */
-  constructor ($rootScope) {
+  constructor ($rootScope, analyticsFactory) {
     this.$rootScope = $rootScope
+    this.analyticsFactory = analyticsFactory
```

Comment:
You also need to add this to the `.module` piece at the bottom (see `stopStartRecurringGifts.modal.component.js`)

Reply:
And you need to import it at the top I believe.

Reply:
Okay went ahead and added that.